examples/python/jupyter_notebook/CipherStash-Getting-Started.ipynb

```diff
@@ -203,7 +266,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%env CS_ENCRYPTION__CLIENT_KEY=a4627031a16b7065726d75746174696f6e9008090a040b000e050c0d020103060f076770325f66726f6da16b7065726d75746174696f6e900d04090e07050f0c030b0608000a01026570325f746fa16b7065726d75746174696f6e90030b0f08070201060d0c05090e00040a627033a16b7065726d75746174696f6e9821031018190a04020e0815181b181c18200c05181d14130900181e0d0712161701181a1818060f11181f0b"
```

Comment:
This was accidentally committed but it's a client key without the corresponding id for a disabled workspace with no important data so I wouldn't consider this to be a big issue (although dataset could be re-enabled)